app/models/post.rb

```diff
@@ -1,3 +1,17 @@
 class Post < ApplicationRecord
   belongs_to :users
```

Comment:
- [ ] In keeping with the ERD relationship, the posts should belong to an `author`, not `users`. Since we do not have an **Author** model, we need to refer Rails to the appropriate class i.e. **User**, like so:

  ```
  belongs_to :author, class_name: 'User'
  ```

- [ ] You also need to implement these changes in your database schema, by renaming the column name from `users_id` to `author_id` using **migrations**. You can refer to this [Stackoverflow answer](https://stackoverflow.com/a/1992045) on how to do that.

  ![oluya-users](https://user-images.githubusercontent.com/36542986/166269096-a9c5d989-8c41-4f81-bed6-f4f67f74cd72.jpg)